### DISCUSSION TITLE
Convert OpenSlide image into an open standard at a similar file size

### DISCUSSION THREAD

**Discussion by coolmule0** [libvips/libvips#3976] [Q&A]
I would like to convert Openslide formats (particularly [svs](https://openslide.org/formats/aperio/) and [mrxs](https://openslide.org/formats/mirax/)) to a more transferable format like tiff. An issue I have is that the generated files are far larger than the initial image. Is there a recommended solution for this?

As an example, a 1.5 GB `.svs` image, with the command 

```
vips openslideload an.svs new.tiff[tile,compression=lzw,bigtiff]
```
becomes 14 GB, around a factor of 10 larger.

Many thanks

**Comment by jcupitt** [comment]
Hello @coolmule0,

SVS and MRXS both usually use JPEG compression, so just use that for your TIFF as well. Maybe Q 85 if you want to be safe. You could also disable chroma subsample if you are very concerned about any losses.

```
$ time vips copy CMU-1.svs x.tif[compression=jpeg,tile,pyramid,Q=85]

real	0m10.302s
user	0m50.133s
sys	0m4.006s
$ ls -l CMU-1.svs x.tif 
-rw-r--r-- 1 john john 177552579 Feb 13  2021 CMU-1.svs
-rw-r--r-- 1 john john 172926272 Jun  4 15:22 x.tif
```
